### DISCUSSION TITLE
Where is the best place to review what has changed between versions?

### DISCUSSION THREAD

**Discussion by jamesshenry** [thomhurst/TUnit#1150] [General]
Currently I'm having a look at merged PRs to see what has changed between frequently released versions. Is there an easier method than this? There doesn't seem to be a CHANGELOG.md

**Comment by thomhurst** [comment]
Yeah apologies. Since I haven't done an official 1.0 release yet I haven't been maintaining a change log. 

I'll raise an issue to make sure I implement one.
